graphics.h

```diff
@@ -0,0 +1,33 @@
+#ifndef GRAPHICS_H_
+#define GRAPHICS_H_
+
+#include <stdint.h>
+#include "text.h"
```

Comment:
Text in separatem Modul

Reply:
Text ist in seperatem Modul. Bisher ist nur die header Datei im repository, wegen der #include-Kette. Ich weiß nicht wie man besser inkludieren kann.

Reply:
In graphics.c werden keine Textfunktionen gebraucht - daher kann der include hier weg. Die Einbindung von Graphics, Text, Cursors und Touch zusammen geschieht in user.**c** (UI-Modul).
Das Cursor-Modul (cursors.c) bindet graphics.h und text.h ein.